pennylane/transforms/adjoint.py

```diff
@@ -131,6 +131,9 @@ def wrapper(*args, **kwargs):
                 if isinstance(new_ops, QuantumTape):
                     new_ops = new_ops.operations

```

Comment:
We should add a TODO here so that we don't forget to remove this :)

Reply:
I forgot what the todo is here ;) Could this be removed in future? It's quite convoluted to see which cases end up here..

Reply:
I don't know either... am I losing my mind 😆